readme.md

```diff
@@ -123,7 +123,7 @@ Each test runner's `stdout` is printed to the console to verify all assertions p
 
 ```
 ~> "ava"   took    598ms  (  ???  )
-~> "jest"  took  1,560ms  (839  ms)
+~> "jest"  took    922ms  (367  ms)
```

Comment:
Can you revert this? I can't take numbers from other machines; especially a single candidate update.

I run everything and update everything from the same machine for consistency.

Reply:
Done.